docs/source/using/installation.rst

```diff
@@ -1,70 +1,98 @@
 Installation
 ============
 
-To install both ``force-bdss`` and the ``force-wfmanager`` applications, first checkout the following
-git repositories
+The BDSS, the Workflow Manager and all plugins can be cloned from the
+`Force 2020 github respositories <https://github.com/force-h2020>`_.
+For the BDSS and Workflow Manager,
 
 .. code-block:: console
 
     git clone https://github.com/force-h2020/force-bdss
     git clone https://github.com/force-h2020/force-wfmanager
+
+This tutorial uses the Enthought-Example and Nevergrad plugins as examples,
+
+.. code-block:: console
+
     git clone https://github.com/force-h2020/force-bdss-plugin-enthought-example
+    git clone https://github.com/force-h2020/force-bdss-plugin-nevergrad
+
+
+Enthought Deployment Manager
+----------------------------
+
+The BDSS, the Workflow Manager and plugins must be installed through the `Enthought Deployment
+Manager (EDM) <https://www.enthought.com/enthought-deployment-manager/>`_, a python environment and
+package manager similar to Anaconda. For new users it is worth examining EDM's
+`documentation <http://docs.enthought.com/edm/>`_.
 
-The last repository is optional, but recommended if you want to practice
-writing plugins.
+To install EDM, follow the instructions specific to your operating system
+`,here <https://docs.enthought.com/edm/installation.html>`_.
 
-If you never installed the Enthought Deployment Manager (EDM), perform the following operations
+Once EDM is installed enter the EDM shell,
 
 .. code-block:: console
 
-    wget https://package-data.enthought.com/edm/rh5_x86_64/1.11/edm_1.11.0_linux_x86_64.sh && bash ./edm_1.11.0_linux_x86_64.sh-b -f -p $HOME
-    export PATH=${HOME}/edm/bin:${PATH}
-    edm install --version 3.6 -y click setuptools
     edm shell
 
-If you instead already have an EDM installation and a default environment, perform the following
+and install the default ``edm`` environment,
 
 .. code-block:: console
 
-    edm shell
     edm install -y click setuptools
 
-Verify that your shell prompt now contains the string "``(edm)``".
-You are now in your default EDM environment, and we assume this environment to be the bootstrap environment.
-The BDSS software will not be installed in this environment, but in a separate one. The following
-commands however must be executed from the bootstrap environment.
+You are now in the ``edm`` environment and your shell prompt is prefixed
+with ``(edm)``.
```

Comment:
I think we could take the opportunity to change these instructions and create a bootstrap environment called `bootstrap`, rather than `edm`.

It removes the `(edm)` environment vs `edm` package manager that has confused users (including me) in the past, and is more in line with our development instructions in other projects.

The installation instruction then just becomes:

`edm install -e bootstrap -y click setuptools`

and the bash shell prompts go to `(bootstrap)`